gslib/tests/util.py

```diff
@@ -172,6 +269,45 @@ def GetUserGroups():
 POSIX_INSUFFICIENT_ACCESS_ERROR = 'Insufficient access with uid/gid/mode for %s'
 
 
+class KmsTestingResources(object):
```

Comment:
If we put this and the below `Authorize` util in their own file, it might be more organized and isolate the dependencies of a dedicated command for test resource setup more (if we don't just use the `kms` command).

Reply:
I think I'll save this for the next stage (implementing the subcommand/flag), just because the implementation of that command will probably suggest a certain location for the file / other things that could be moved there.